core/arch/arm/plat-d06/main.c

```diff
@@ -4,7 +4,17 @@
  * Copyright (c) 2022, Huawei Technologies Co., Ltd
  */
 #include <console.h>
+#include <drivers/lpc_uart.h>
+#include <platform_config.h>
+
+static struct lpc_uart_data console_data;
```

Comment:
This probably needs `__nex_bss` for virtualization, but I am wondering what our policy should be in that regard: if the platform isn't tested with `CFG_VIRTUALIZATION=y` then it might be OK to leave this as is. @jenswi-linaro @lorc what do you think? This reminds me of the FF-A vs. pager discussion (commit 593b94ee5eb0).

Personally I would still recommend adding `__nex_bss` here.

Reply:
I'm fine with either way. It's currently not needed, and will perhaps never be needed.